File: MyAnimeList/src/main/java/com/example/myanimelist/views/models/Presentation.java
package com.example.myanimelist.views.models;

import javafx.beans.property.SimpleStringProperty;
import javafx.beans.property.StringProperty;

public class Presentation {

    final StringProperty title;
    final StringProperty titleEnglish;
    final StringProperty img;

    public Presentation(String title, String titleEnglish, String img) {
        this.img = new SimpleStringProperty(img);
        this.title = new SimpleStringProperty(title);
        this.titleEnglish = new SimpleStringProperty(titleEnglish);
    }

    public String getTitle() {
        return title.get();
    }

    public void setTitle(String title) {
        this.title.set(title);
    }

    public StringProperty titleProperty() {
        return title;
    }

    public String getTitleEnglish() {
        return titleEnglish.get();
    }

    public void setTitleEnglish(String titleEnglish) {
        this.titleEnglish.set(titleEnglish);
    }

    public StringProperty titleEnglishProperty() {
        return titleEnglish;
    }

    public String getImg() {
        return img.get();
    }

    public void setImg(String img) {
        this.img.set(img);
    }

    public StringProperty imgProperty() {
        return img;
    }
}

File: MyAnimeList/src/main/kotlin/com/example/myanimelist/views/models/Presentation.kt
package com.example.myanimelist.views.models

import javafx.beans.property.SimpleStringProperty
import javafx.beans.property.StringProperty

class Presentation(title: String?, titleEnglish: String?, img: String?) {
    val title: StringProperty
    val titleEnglish: StringProperty
    var img: StringProperty

    init {
        this.img = SimpleStringProperty(img)
        this.title = SimpleStringProperty(title)
        this.titleEnglish = SimpleStringProperty(titleEnglish)
    }

    fun getTitle(): String {
        return title.get()
    }

    fun setTitle(title: String) {
        this.title.set(title)
    }

    fun titleProperty(): StringProperty {
        return title
    }

    fun getTitleEnglish(): String {
        return titleEnglish.get()
    }

    fun setTitleEnglish(titleEnglish: String) {
        this.titleEnglish.set(titleEnglish)
    }

    fun titleEnglishProperty(): StringProperty {
        return titleEnglish
    }

    fun getImg(): String? {
        return img.get()
    }

    fun setImg(img: String) {
        this.img.set(img)
    }

    fun imgProperty(): StringProperty {
        return img
    }
}